src/app/pages/RuntimeTransactionDetailPage/index.tsx

```diff
@@ -201,6 +221,60 @@ export const RuntimeTransactionDetailView: FC<{
             </>
           )}
 
+          <dt>{t('common.recentTransactions')}</dt>
+          <dd>
+            <Box sx={{ display: 'block', overflowX: 'auto' }}>
+              {transfers?.map((transfer, i) => {
+                const params = transfer.evm_log_params
+                if (!params) return null
+
+                const from = params.find(p => p.name === 'from')?.value as string | undefined
+                const to = params.find(p => p.name === 'to')?.value as string | undefined
+                const amount = params.find(p => p.name === 'value')?.value as string | undefined
+                const symbol = transfer.evm_token?.symbol
+
+                return (
+                  <Box
+                    key={i}
+                    sx={{
+                      display: 'flex',
+                      alignItems: 'center',
+                      gap: 2,
+                      mb: 2,
+                      minWidth: '100%',
+                      whiteSpace: 'nowrap',
+                    }}
+                  >
+                    <TokenTypeTag tokenType={transfer.evm_token?.type} />
+                    <Typography variant="body2">{t('common.from')} </Typography>
+                    {from ? <AccountLink scope={scope} address={from} alwaysTrim /> : '?'}
+
+                    <Typography variant="body2">{t('common.to')}</Typography>
+                    {to ? <AccountLink scope={scope} address={to} alwaysTrim /> : '?'}
+
+                    <Typography variant="body2">
+                      {t('common.for')} {amount} {symbol}
```

Comment:
Amount should be formatted, like we do elsewhere.
It probably doesn't need to be precise, so maybe RoundedBalance compactLargeNumbers.